handlers/__init__.py

```diff
@@ -0,0 +1,134 @@
+from abc import ABC, abstractmethod
```

Comment:
I think this file might be best moved to something like `notification.py` outside of this directory. It does very little common `__init__.py` tasks (initialising the package, exporting methods and classes). Thoughts?